app/packages/relay/src/resolve.ts

```diff
@@ -0,0 +1,9 @@
+export default <T extends unknown>(module: T): T => {
+  // @ts-ignore
```

Comment:
For some of these random fixes can you add a comment so we remember why these things exist?

Reply:
+1